### ISSUE TITLE
Please support global function handler

### ISSUE THREAD

**Issue by dlydiard** [SnappyTutorials/aws-sam-webpack-plugin#13]
```
Globals:
  Function:
    Runtime: nodejs10.x
    Handler: app.handler
    MemorySize: 512
    Timeout: 30
```

Error: Endpoint is missing a Handler

**Comment by dlevs** [comment]
This looks related to this PR:
https://github.com/SnappyTutorials/aws-sam-webpack-plugin/pull/15

**Comment by buggy** [comment]
This has been fixed and will be released with 0.5.0
